src/PrestaShopBundle/Controller/Admin/Improve/International/CurrencyController.php

```diff
@@ -205,6 +251,135 @@ public function toggleStatusAction($currencyId)
         return $this->redirectToRoute('admin_currencies_index');
     }
 
+    /**
+     * Updates exchange rates.
+     *
+     * @AdminSecurity(
+     *     "is_granted('update', request.get('_legacy_controller'))",
+     *     redirectRoute="admin_currencies_index",
+     *     message="You do not have permission to edit this."
+     * )
+     * @DemoRestricted(redirectRoute="admin_currencies_index")
+     *
+     * @return RedirectResponse
+     */
+    public function updateExchangeRatesAction()
```

Comment:
action says it "updates" while command says it "refreshes", i think you choose one.